pkg/ctl/delete/cluster.go

```diff
@@ -103,11 +104,23 @@ func doDeleteCluster(rc *cmdutils.ResourceCmd) error {
 	}
 
 	{
+
+		logger.Info("cleaning up LoadBalancer services")
+		if err := ctl.GetCredentials(cfg); err != nil {
+			return err
+		}
+		client, err := ctl.NewClient(cfg, false)
+		if err != nil {
+			return err
+		}
+		elb.CleanupLoadBalancers(ctl.Provider.ELB(), ctl.Provider.ELBV2(), client)
```

Comment:
We should prepend it to tasks actually, but you will need some of the changes from #847 for that, I can pick those onto another PR if you like, or we refactor later on.